README.md

```diff
@@ -17,6 +17,25 @@ error prone plumbing.
 
 *nom will happily take a byte out of your files :)*
 
+<!-- toc -->
```

Comment:
Are these comments somehow interpreted by something or are they just for human orientation?

Reply:
Just for human orientation, 
I took the inspiration from pytorch's README, I wasn't sure if these comments were necessary though